### ISSUE TITLE
remove 'Back' btn from default focus flow

### ISSUE THREAD

**Issue by sydneywalcoff** [hackforla/expunge-assist#1237]
### Overview

Accessibility for users navigating by keyboard states that we should create a _logical_ path for the focus to follow. While navigating the letter generator, the focus will move from the main content to the 'back' button before moving to 'next' button. 

I would like to take the 'back' button out of the default focus flow. The focus should then move from the main content to the 'next' button. The user should be able to access the back button by moving backward from the 'next' button.

### Action Items

- [ ] adjust the focus flow so that the 'back' button isn't part of the default focus flow

### Resources/Notes

https://www.accessibility-developer-guide.com/

**Comment by davidwiese** [comment]
Update: Should have this completed by Friday 3/22/24
